compiler_opt/rl/train_bc.py

```diff
@@ -35,29 +35,30 @@
 
 from typing import Dict
 
-_ROOT_DIR = flags.DEFINE_string(
-    'root_dir', os.getenv('TEST_UNDECLARED_OUTPUTS_DIR'),
-    'Root directory for writing logs/summaries/checkpoints.')
-_DATA_PATH = flags.DEFINE_multi_string(
-    'data_path', [],
-    'Path to TFRecord file(s) containing training data. Skip training and dump'
-    'an untrained model with random weights (for testing purpose) if '
-    'unspecified.')
-_GIN_FILES = flags.DEFINE_multi_string(
-    'gin_files', [], 'List of paths to gin configuration files.')
-_GIN_BINDINGS = flags.DEFINE_multi_string(
-    'gin_bindings', [],
-    'Gin bindings to override the values set in the config files.')
+if __name__ == '__main__':
```

Comment:
do you still need this and changing the train_eval interface after switching to subprocess?

Reply:
Good catch. I thought I reverted all these changes previously, but apparently I didn't. Fixed now.